### ISSUE TITLE
Baseline file option not compatible with multi module project

### ISSUE THREAD

**Issue by raymondboswel** [detekt/detekt#5712]
## Expected Behavior
Setting the baseline file location should either merge all baseline files from subprojects, or documentation should make it clear that each subproject will overwrite the baseline file.

## Observed Behavior
Running ./gradlew detektBaselineMain in a multi module project results in a baseline file that clearly doesn't contain all the issues. After some head scratching and looking through existing issues the penny dropped that each module was overwriting the file.

I think adding a note in the docs should be a sufficient fix for this problem.

**Comment by BraisGabin** [comment]
Could you create a PR making the documentation more clear to avoid future confusions?

We don't support an unified baseline out of the box. The idea is to have a baseline for each module.

**Comment by raymondboswel** [comment]
Sure I'll give it a shot. What confused me, was that I set up detekt for our team about a year ago (with no updates in the mean time), and up till now it put everything in one big baseline file. We recently encountered a bug due to unreachable code, which made me realize that we weren't running in Type Resolution mode. After refactoring our config to enable TR, the baseline behaviour changed, though it could also just be the version update that changed the behaviour :shrug:
